test/tests/functional/pbs_test_tpp.py

```diff
@@ -161,6 +160,8 @@ def submit_job(self, set_attr=None, exp_attr=None, job=False,
         if resv_job:
             if 'ATTR_inter' in set_attr:
```

Comment:
I think you don't need this because there no attr named 'ATTR_inter' in pbs, and based on below `elif` here you need `ATTR_inter` (without single quote) (and in pbs `ATTR_inter` == `'interactive'`), and you don't need `elif` part.